understanding/20/media-alternative-prerecorded.html

```diff
@@ -12,11 +12,12 @@ <h1>Understanding Media Alternative (Prerecorded)</h1>
    <section id="brief">
       <h2>In brief</h2>
       <dl>
-         <dt>Objective</dt><dd>Precorded videos can be understood by more people</dd>
-         <dt>Author task</dt><dd>Provide a text equivalent for all content in videos</dd>
+         <dt>Goal</dt><dd>Precorded videos can be understood by more people.</dd>
+         <dt>What to do</dt><dd>Provide a text equivalent for all content in videos.</dd>
+         <dt>Why it's important</dt><dd>More people, including those who are deaf-blind, can better understand the content at their own pace.</dd> 
```

Comment:
@rainbreaw commented
> Consider including people with cognitive disabilities.

Reply:
I would like to call out deaf-blind users exclusively here, since it is the only SC specifically listing that need AFAIK. The wording does not exclude other users.

Reply:
@GreggVan gave +1 for Rain's suggestion